### ISSUE TITLE
cf_logit request!

### ISSUE THREAD

**Issue by lu-416** [SAI990323/DecodingMatters#1]
I tried using the cf_logits from my own trained SASRec model, but the results were not satisfactory. Could you provide the original logits file and the related parameter settings (alpha)?
